app/assets/javascripts/order.js

```diff
@@ -4,8 +4,12 @@ $(document).ready(function() {
   $("#new_order").submit(function(event) {
     var $form = $(this);
     $form.find("button").prop("disabled", true);
-    Stripe.card.createToken($form, stripeResponseHandler);
-    return false;
+    if ($("#card_number").length) {
+      Stripe.card.createToken($form, stripeResponseHandler);
+      return false;
+  } else {
+      true;
```

Comment:
1. False because form submits inside stripeResponseHandler method so I don't want to submit it again. True just moves on through the function, but since it's at its end, it doesn't do anything. I need it though to establish an else path. Thoughts are welcomed!
2. Worked without return true but added it to be consistent with return false. Just a stylistic preference, yes?